aiidalab_launch/profile.py

```diff
@@ -73,6 +74,14 @@ def __post_init__(self):
     def container_name(self) -> str:
         return f"{CONTAINER_PREFIX}{self.name}"
 
+    def parse_custom_mount(self, custom_mount: str) -> tuple[Path, Path]:
+        # TODO: Add more error handling
+        source, target = custom_mount.split(":")
```

Comment:
I am using the Docker notation `source:target` here, but maybe there is a better solution?